test/suites/ACID.m

```diff
@@ -334,6 +335,7 @@
 function [stat] = contourPenny()
   stat.description = 'Contour plot of a US\$ Penny.';
   stat.unreliable = isMATLAB('>=',[8,4]);
+  % see #604; contour() produces inconsistent output
```

Comment:
Can you leave a `TODO` or `FIXME` there to make sure that we come back to those?